devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy

```diff
@@ -222,20 +211,103 @@ def poll() {
 }
 
 def refresh() {
-	delayBetween([
-		zwave.switchBinaryV1.switchBinaryGet().format(),
-		zwave.meterV2.meterGet(scale: 0).format(),
-		zwave.meterV2.meterGet(scale: 2).format()
-	], 500)
+	log.debug "refresh()"
+	encapSequence([
+		zwave.switchBinaryV1.switchBinaryGet(),
+		meterGet(scale: 0),
+		meterGet(scale: 2)
+	])
 }
 
 def configure() {
-	zwave.manufacturerSpecificV2.manufacturerSpecificGet().format()
+	log.debug "configure()"
+	def result = []
+
+	log.debug "Configure zwaveInfo: "+zwaveInfo
+
+	if (zwaveInfo.mfr == "0086") {	// Aeon Labs meter
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 80, size: 1, scaledConfigurationValue: 2)))	// basic report cc
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 101, size: 4, scaledConfigurationValue: 1)))	// report power in watts
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 111, size: 4, scaledConfigurationValue: 300)))	 // every 5 min
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 102, size: 4, scaledConfigurationValue: 2)))	// report energy in kWh
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 112, size: 4, scaledConfigurationValue: 300)))	 // every 5 min
+	}
+	result << response(encap(meterGet(scale: 0)))
+	result << response(encap(meterGet(scale: 2)))
+	result
 }
 
 def reset() {
-	return [
-		zwave.meterV2.meterReset().format(),
-		zwave.meterV2.meterGet(scale: 0).format()
-	]
+	encapSequence([
+		meterReset(),
+		meterGet(scale: 0)
+	])
+}
+
+def meterGet(scale)
```

Comment:
Would be better for this to be the value of the scale as an integer, not the entire param map. Or at least change the name of this variable so it's accurate.